warp10/src/main/java/io/warp10/hadoop/Warp10RecordReader.java

```diff
@@ -171,6 +189,14 @@ public void initialize(InputSplit split, TaskAttemptContext context)
           conn.setRequestProperty(Warp10InputFormat.HTTP_HEADER_DEDUP, dedup);
         }
 
+        //
+        // Position attributes
+        //
+
+        for (Entry<String,String> attr: attributes.entrySet()) {
+          conn.setRequestProperty(Warp10InputFormat.HTTP_HEADER_ATTR_PREFIX + attr.getKey(), attr.getValue());
```

Comment:
Could it be a good idea here to raise a runtime exception if the attribute contains a dot ? It can avoid painful debugging later.